src/pages/about.js

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import styled from "styled-components";
+import { graphql } from "gatsby";
+import "../styles/fonts.scss";
+import Layout from "../components/Layout";
+import ENVISION from "../images/svg/ENVISION.svg";
+import HeroImage from "../images/herogradient.png";
+import Envision from "../images/ENVISION.png";
+import colors from "../styles/colors";
+import fonts from "../styles/font_names";
+import {
+  AboutContainer,
+  AboutHero,
+  AboutHeroParagraph,
+  AboutHeroText,
+  ParagraphColumn,
+} from "../styles/About.styles";
+
+const AboutPage = ({ data }) => {
+  const aboutQuery = data.prismicAbout.data;
+
+  return (
+    <Layout>
+      <AboutContainer>
+        <AboutHero>
+          <AboutHeroText>
+            {aboutQuery.about_header.split(" ").map((word, id) => (
+              <span key={id} className={id == 2 && "orange"}>
```

Comment:
instead of adding a `className` here since you aren't using classes anywhere else on the site, can you make this a styled component and conditionally pass a prop to render it w the orange color?